test/command/cmd.go

```diff
@@ -204,35 +214,53 @@ func (s *Starter) KillAll() {
 			continue
 		}
 		if v, ok := cmd.Value.(*exec.Cmd); ok {
-			s.kill(v)
+			err := s.kill(v)
```

Comment:
I think we should continue even if failed to kill some cmd. And we can return the errors with a slice. WDYT?

Reply:
use a slice?
It will change a lot of code.
or keep it as same as before, do not handle this error.WDYT

Reply:
IMOP, that's OK. 😄